<!--edit by lichunjiing 2020_12_23 -->
<launch>
  <param name="/use_sim_time" value="false" />
  <arg name="rviz" default="true" />

  <arg name="cfg_file_path" default="/home/alorli/WORK_SPACE_LCJ/catkin_ws/src/lidar_mapping/cfg" />

  <node name="map_building_visualization_node"  pkg="lidar_mapping"  type="map_building_visualization_node" output="screen">
    <param name="cfg_file_path" value="$(arg cfg_file_path)" />
  </node>

  <group if="$(arg rviz)">
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_mapping)/rviz/map_building_visualization.rviz" />
  </group>

</launch>
